parallel.cabal

```diff
@@ -24,12 +24,11 @@ tested-with:
   GHC == 8.2.2
   GHC == 8.0.2
   GHC == 7.10.3
-  -- Drop these old GHCs from CI:
-  -- GHC == 7.8.4
-  -- GHC == 7.6.3
-  -- GHC == 7.4.2
-  -- GHC == 7.2.2
-  -- GHC == 7.0.4
+  GHC == 7.8.4
+  GHC == 7.6.3
+  GHC == 7.4.2
+  GHC == 7.2.2
+  GHC == 7.0.4
```

Comment:
So we are reintegrating them in CI? They are **very** old

Reply:
As long as it works without changes, I see no reason not to. I expect the testing libraries to not support such old versions anyway though.

Reply:
Apparently it won't, if we use `nothunks` in the test suite. :)

Reply:
Yeah, now we only test GHC >= 8.6.